ports/winit/browser.rs

```diff
@@ -31,6 +31,8 @@ use tinyfiledialogs::{self, MessageBoxIcon, OkCancel, YesNo};
 
 pub struct Browser<Window: WindowPortsMethods + ?Sized> {
     current_url: Option<ServoUrl>,
+    current_url_taken: bool,
```

Comment:
Hrm. I think maybe it would be a little simpler to just store a stringified version of the URL here when it changes. Then you could only do the conversion when the URL actually change and pass and expose a reference to a getter.

Reply:
great idea, that makes a lot more sense! done in b01b10c591801be4736ccd501cfa292c9a361bc7